Documentation/network/servicemesh/ingress.rst

```diff
@@ -82,23 +86,20 @@ Supported Ingress Annotations
      - ``LoadBalancer``
    * - ``ingress.cilium.io/insecure-node-port``
      - | The NodePort to use for the HTTP Ingress.
-       | Applicable only if ``ingress.cilium.io/service-type`` is ``NodePort``. If unspecified, a random
+       | Applicable only if ``ingress.cilium.io/service-type``
+       | is ``NodePort``. If unspecified, a random
        | NodePort will be allocated by kubernetes.
      - unspecified
    * - ``ingress.cilium.io/secure-node-port``
      - | The NodePort to use for the HTTPS Ingress.
-       | Applicable only if ``ingress.cilium.io/service-type`` is ``NodePort``. If unspecified, a random
+       | Applicable only if ``ingress.cilium.io/service-type``
+       | is ``NodePort``. If unspecified, a random
        | NodePort will be allocated by kubernetes.
      - unspecified
-   * - ``ingress.cilium.io/http-host-port``
-     - | The port to use for the HTTP listener (HTTP and HTTPS) on the host network.
-       | Applicable only for dedicated Ingress and if hostnetwork mode is enabled for IngressController.
-       | If unspecified, the default ports (80/443) are used.
-     - unspecified
-   * - ``ingress.cilium.io/tls-passthrough-host-port``
-     - | The port to use for the TLS passthrough listener on the host network.
-       | Applicable only for dedicated Ingress and if hostnetwork mode is enabled for IngressController.
-       | If unspecified, the default port (443) are used.
```

Comment:
@mhofstetter I assume `ingress.cilium.io/host-port` doesn't have this default port `443` anymore, right?

Reply:
@PhilipSchmid thanks for pointing this out. i realized i still don't have this right in the related PR.

in case of hostnetwork all "port types" (HTTP, HTTPS & TLS Passthrough) need to default to the same port. Still to be defined what the default should be - whether `80`, `443` or something completely different (maybe even in the unprivileged port range?). Using `80` or `443` may imply that this is used for a specific purpose (HTTP, HTTPS or TLS passthrough) - which it doesn't - as it's used for all cases.

Reply:
In any way - there's a high chance for port clashes if the user doesn't provide specific ports for the dedicated ingresses (as they all would use the same default port)

Reply:
How about not specifying any default for `ingress.cilium.io/host-port` at all and then if the user doesn't configure it to a unique, still available port, we could deny serving it via hostNetwork mode (and throw a warning)?

Reply:
as discussed, this might be something for the future. It's not that easy to check for an available port, as this would happen on the Cilium Operator whereas the Listeners are created on the Cilium Agents (with potentially different ports already bound on the different nodes)